src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.html

```diff
@@ -0,0 +1,300 @@
+<cd-loading-panel *ngIf="loading && !error"
+                  i18n>Loading configuration...</cd-loading-panel>
+<cd-alert-panel type="error"
+                *ngIf="loading && error"
+                i18n>The configuration could not be loaded.</cd-alert-panel>
+
+<div class="cd-col-form"
+     *ngIf="!loading && !error">
+  <ng-container [ngSwitch]="step">
+    <!-- Configuration step -->
+    <div *ngSwitchCase="1">
+      <form name="form"
+            #formDir="ngForm"
+            [formGroup]="configForm"
+            novalidate>
+        <div class="card">
+          <div class="card-header"
+               i18n>Step {{ step }} of 2: Telemetry report configuration</div>
+          <div class="card-body">
+            <p i18n>The telemetry module sends anonymous data about the cluster back to the Ceph developers
```

Comment:
Small textual improvement suggestion:
```suggestion
            <p i18n>The telemetry module sends anonymous data about this Ceph cluster back to the Ceph developers
```